src/features/goblins/bank/components/Withdraw.tsx

```diff
@@ -126,28 +101,21 @@ export const Withdraw: React.FC<Props> = ({ onClose }) => {
     onClose();
   };
 
-  if (isLoading) {
-    return <span className="loading">Loading</span>;
-  }
-
-  if (jiggerState) {
-    return (
-      <Jigger
-        onClose={onClose}
-        status={jiggerState.status}
-        verificationUrl={jiggerState.url}
-      />
-    );
-  }
-
-  const isBlacklisted = authState.context.blacklistStatus !== "OK";
+  const proovePersonhood = async () => {
+    goblinService.send("PROVE_PERSONHOOD");
+    onClose();
+  };
 
-  if (isBlacklisted) {
+  if (!goblinState.context.verified) {
     return (
-      <div className="p-2 text-sm">
-        Withdrawing is temporarily restricted while your humanity is being
-        verified. Thanks for your patience!
-      </div>
+      <>
+        <p className="text-sm p-1 m-1">
+          You must verify your identity to use this feature.
```

Comment:
This feels a bit KYC.

Consider the following: "Proof of humanity is needed for this feature. Please take a quick selfie."